tally_ho/apps/tally/forms/tally_form.py

```diff
@@ -35,6 +35,14 @@ def __init__(self, *args, **kwargs):
         super(TallyForm, self).__init__(*args, **kwargs)
         self.fields['name'].widget.attrs.update({'class': 'form-control'})
 
+    def clean(self):
```

Comment:
i think you should convert this into a util functioon

Reply:
Done